test/integration/cluster/test_cluster_show.py

```diff
@@ -261,17 +319,49 @@ def test_cluster_show_config_etcd_no_key(tt_cmd, tmpdir_with_cfg, etcd):
     )
     show_output = instance_process.stdout.read()
 
-    expected = (r"   ⨯ failed to collect a configuration: " +
-                "a configuration data not found in etcd for key \"/prefix/config/foo\"")
+    expected = (
+        r"   ⨯ failed to collect a configuration: "
+        + f'a configuration data not found in {storage_name} for key "/prefix/config/foo"'
+    )
     assert expected in show_output
 
 
-def test_cluster_show_config_etcd_no_auth(tt_cmd, tmpdir_with_cfg, etcd):
+@pytest.mark.parametrize(
+    "instance_name, err_msg",
+    [
+        pytest.param(
+            "etcd",
+            "   ⨯ failed to collect a configuration: "
+            + "failed to fetch data from etcd: etcdserver: user name is empty",
+        ),
+        pytest.param(
+            "tcs",
+            "   ⨯ failed to collect a configuration: "
+            + "failed to fetch data from tarantool: Execute access to function "
+            + "'config.storage.get' is denied for user"
+        ),
+    ],
+)
+def test_cluster_show_config_no_auth(
+    tt_cmd, tmpdir_with_cfg, instance_name, request, err_msg, fixture_params
+):
+    if instance_name == "tcs":
+        if utils.is_tarantool_less_3() or not is_tarantool_ee():
+            pytest.skip()
+        fixture_params["path_to_cfg_dir"] = os.path.join(os.path.dirname(
+                                            os.path.abspath(__file__)), "test_tcs_app")
+        fixture_params["connection_test"] = True
+        fixture_params["connection_test_user"] = "client"
+        fixture_params["connection_test_password"] = "secret"
+        fixture_params["instance_name"] = "instance-001"
+        fixture_params["instance_host"] = "localhost"
+        fixture_params["instance_port"] = "3303"
```

Comment:
Perhaps there is a way to avoid copy-pasting these lines everywhere in the module? For example, declare `fixture_tcs_params` at the module level.
```suggestion
        for k, v in fixture_tcs_params:
             fixture_params[k] = v
```